BoDonkey/typography-extension
# Vue component refactor

## Summary

*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Closes #[issue number]"*
This PR attempts to optimize the logic of the Character Control module components by moving some of the duplicated logic to a mixin. Additionally, the package name was corrected in the `package.json` file to match the new repo name. This code should be looked at for functionality and with an eye toward being used in a tutorial. So, if it  maintains function, but is not a good code practice, it should be flagged (or outright rejected).

## What are the specific steps to test this change?

*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly

1. Install the modules in an A3 project.
2. Add the `characterCount` tool and insert to a rich-text-widget-configuration. 
3. Check for functionality by the toolbar button or insert menu item.
## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
